Automattic/vip-go-node
# Add Redis helper

Round 3!

Goal: Make it easier for clients to use Redis with VIP Go configurations.

We're adding a helper library that Node apps can use to instantiate a Redis client compatible with VIP Go.

This is a combination of the work from @pandah3 and @elazzabi (#125 + #188).

Note; ioredis is set as a devDependency and we require users to install it explicitly. This is to avoid adding bloat to the main library for all users (see https://github.com/Automattic/vip-go-node/pull/188#issuecomment-577794666)

Fixes #110

## Steps to Test

- Run redis-server
- Run REDIS_MASTER=[HOST]:[PORT] node ./src/redis/index.js
- Should log message Connected to Redis client

Also: `docker-compose up && npm run test`
